sass/components/_content_project.scss

```diff
@@ -0,0 +1,10 @@
+/**
+ * @file
+ * This file is used to styles the Project content page.
+ */
+#project-stakeholder {
+  .ecl-description-list__definition {
```

Comment:
It is a workaround of https://citnet.tech.ec.europa.eu/CITnet/jira/browse/FRONT-1279

Reply:
Let's put a comment here to remind ourselves to remove this when that issue is fixed:
```
// Remove this when FRONT-1279 is fixed.
```
Also please add the same comment on _description-list.scss, I forgot about it.